troubleshooting/known_issues/nmn_dns_r_records_missing.md

```diff
@@ -0,0 +1,218 @@
+# iSCSI SBPS projection over NMN will fail if DNS "A" (address) records for NMN are not created
+
+## Issue Description
+
+iSCSI based boot content projection, which is also known as "Scalable Boot Content Projection" (SBPS), for `rootfs` and `PE` images is
+supported in CSM version CSM 1.6.0 and later. On a customer system, using `CSM-1.6.0` and later, DNS SRV "A" records
+for NMN might not be created due to trailing `CRLFs`(Carriage Return Line Feed) with NMN IPs.
+
+## Issue Identification
+
+This issue can be identified by the following symptoms:
+
+On any worker node, use one of the following commands to observe that the DNS "SRV" "A" records are missing for NMN
+
+### Using `powerdns` command
+
+```bash
+ncn-w001:~ # kubectl -n services exec -it deployment/cray-dns-powerdns -- sh -c 'pdnsutil list-all-zones | xargs -n1 pdnsutil list-zone | grep iscsi'
+_sbps-hsn._tcp.drax.hpc.amslabs.hpecorp.net     3600    IN      SRV     1 0 3260 iscsi-server-id-004.hsn.drax.hpc.amslabs.hpecorp.net.

```

Comment:
Break this and the dig command snippets into command & example blocks without the prompts (e.g. no `ncn-w001:~ #`) for easier copy-paste-and-edit.